brain/compartmentReport.h

```diff
@@ -77,8 +89,13 @@ class CompartmentReport
     BRAIN_API CompartmentReport(const URI& uri);
     BRAIN_API ~CompartmentReport();
 
+#ifndef DOXYGEN_TO_BREATHE
     /** @return the metadata of the report */
     BRAIN_API const CompartmentReportMetaData& getMetaData() const;
+#else
+    /** @return the metadata of the report */
```

Comment:
You have forgotten to add the specification of the dictionary returned (key names and semantics)